api/opencl_platform_layer.asciidoc

```diff
@@ -1167,18 +1167,20 @@ include::{generated}/api/version-notes/CL_DEVICE_EXTENSIONS.asciidoc[]
         *cl_khr_local_int32_extended_atomics*
 
         Additionally, the following Khronos extension names must be returned
-        by all devices that support OpenCL 1.2:
+        by all devices that support OpenCL 1.2 when the optional feature is
+        supported:
 
         *cl_khr_fp64* (if double precision is supported)
 
         Additionally, the following Khronos extension names must be returned
         by all devices that support OpenCL 2.0, OpenCL 2.1, or OpenCL 2.2.
-        Note that these Khronos extension names are *not* required for
-        devices supporting OpenCL 3.0:
+        For devices that support OpenCL 3.0, these extension names must only
+        be returned when the optional feature is supported:
```

Comment:
Being pedantic: maybe we want something along the lines of "when and only when".

Reply:
Good clarification, fixed.

Reply:
Looks good.